### ISSUE TITLE
/simulation-frame: srw.extract_report_data(): KeyError: 'models'

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#590]
During a partially coherent run. I suspect the file was not written completely.  Stack trace below. I was able to repeat it. Happened at a different frame the second time.

```
10.10.10.1 - - [05/Jan/2017 00:52:03] "GET /simulation-frame/srw*XdV1QagN*multiElectronAnimation**3*1483577480716876 HTTP/1.1" 500 -
Error on request:
Traceback (most recent call last):
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in execute
    application_iter = app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1994, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/beaker/middleware.py", line 156, in __call__
    return self.wrap_app(environ, session_start_response)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 634, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 422, in app_simulation_frame
    response = _json_response(template.get_simulation_frame(run_dir, data, model_data))
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/template/srw.py", line 487, in get_simulation_frame
    return extract_report_data(str(run_dir.join(get_filename_for_model(data['report']))), data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/template/srw.py", line 186, in extract_report_data
    width_pixels = int(model_data['models']['simulation']['intensityPlotsWidth'])
KeyError: 'models'
```

**Comment by robnagler** [comment]
Tabulated Undulator Example

**Comment by mrakitin** [comment]
I don't see this problem on alpha. Did you try to discard changes and rerun?

**Comment by mrakitin** [comment]
My bad, I did not see it was for the partially-coherent calculation. I got `502 Bad Gateway` for that as well.

**Comment by mrakitin** [comment]
The reason is that the `model_data` is
```python
{'modelName': u'multiElectronAnimation', 'animationArgs': u'', 'simulationType': u'srw', 'startTime': u'1483740762745250', 'report': u'multiElectronAnimation', 'simulationId': u'thJxD2eb', 'frameIndex': u'6'}
```
so it does not contain the fields from schema.

**Comment by mrakitin** [comment]
I'm debugging it.

**Comment by mrakitin** [comment]
Fixed in  	9726f29.
